index.js

```diff
@@ -103,7 +105,12 @@ var deps = moduleDeps({
           }
         }
 
-        cb(null, /^\//.test(file) ? file : null)
+        if ((file = /^\//.test(file) ? file : null) && !argv['disable-hashing']) {
+          fs.readFile(file, function (err, contents) {
+            var h = crypto.createHash('sha1').update(contents).digest('hex')
+            cb(null, map[h] || (map[h] = file))
```

Comment:
right here, if the hash has been saved into the map, I return the location of the last file with its exact contents instead of the resolved one..

if the hash doesn't exist into the map, save the resolved file's location into the map and rueturn it to the callback